modules/govuk/manifests/node/s_elasticsearch.pp

```diff
@@ -14,11 +17,11 @@
   }
 
   class { 'govuk_elasticsearch':
-    cluster_hosts        => ['elasticsearch-1.backend:9300', 'elasticsearch-2.backend:9300', 'elasticsearch-3.backend:9300'],
+    cluster_hosts        => $cluster_hosts,
     cluster_name         => 'govuk-production',
     heap_size            => "${es_heap_size}m",
     number_of_replicas   => '1',
-    minimum_master_nodes => '2',
+    minimum_master_nodes => $minimum_master_nodes,
```

Comment:
Does this mean we'll change the minimum number of nodes in production from 2 to 1?